### ISSUE TITLE
fabric8-ui builds are failing

### ISSUE THREAD

**Issue by nimishamukherjee** [openshiftio/openshift.io#2565]
Impact: fabric8-planner PRs cannot be merged as we cannot run the integrated UI

Fix - https://github.com/fabric8-ui/fabric8-ui/pull/2615 needs to get merged.

**Comment by arunkumars08** [comment]
This PR has the fix that takes care of the build - https://github.com/fabric8-ui/fabric8-ui/pull/2634

Currently, just the fix from the PR - https://github.com/fabric8-ui/fabric8-ui/pull/2615 has been moved to a new PR to unblock the flow.

@nimishamukherjee

**Comment by arunkumars08** [comment]
Once the build goes through for the new PR, it will be merged in.

**Comment by nimishamukherjee** [comment]
Thanks @arunkumars08

**Comment by nimishamukherjee** [comment]
https://github.com/fabric8-ui/fabric8-ui/pull/2634 merged.

Planner PRs are working now https://jenkins.cd.test.fabric8.io/blue/organizations/jenkins/fabric8-ui%2Ffabric8-planner/detail/PR-2487/5/pipeline/

Closing this issue.
